iib/web/static/api_v1.yaml

```diff
@@ -709,6 +748,49 @@ components:
               example: openshift-operators
           required:
             - from_bundle_image
+    RequestMergeIndexImage:
+      type: object
+      properties:
+        binary_image:
+          type: string
+          example: 'quay.io/operator-framework/upstream-registry-builder:v1.5.9'
+        deprecation_list:
+          type: array
+          items:
+            type: string
+          description: >
+            List of bundles (specified by hash) to be deprecated from the new index image.
+          example:
+            - 'quay.io/user/openshift-bundle-example-container@sha256:29e4531e4625653'
+        source_from_index:
+          type: string
+          description: >
+            Index image that will be used as a base for the new index image.
+          example: 'quay.io/iib-stage/iib:4.5'
+        target_index:
+          type: string
+          description: >
+            Pull specification of the index image whose contents are to be cleaned and updated.
```

Comment:
...whose bundles from which are going to be added to the content of from_index. The resulting index image will use the ocp_version label of this index image.